java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3101,12 +3101,11 @@ public T xgroupDelConsumer(@NonNull String key, @NonNull String group, @NonNull
      *     will be read. Use the special id of <code>{@literal Map<String, Map<String, String[][]>>}
      *     </code> to receive only new messages.
      * @param group The consumer group name.
-     * @param consumer The newly created consumer.
+     * @param consumer The consumer name.
      * @return Command Response - A <code>{@literal Map<String, Map<String, String[][]>>}</code> with
      *     stream keys, to <code>Map</code> of stream-ids, to an array of pairings with format <code>
      *     [[field, entry], [field, entry], ...]<code>.
-     *     Returns <code>null</code> if the consumer group does not exist. Returns a <code>Map</code>
-     *     with a value of code>null</code> if the stream is empty.
+     *     Returns <code>null</code> if the consumer group does not exist.
```

Comment:
Same, if the group doesn't exist a NOGROUP error will occur, lets do this instead:
```suggestion
    *      Returns <code>null</code> if there is no stream that can be served.
```